server/server-kv.h

```diff
@@ -1,27 +1,13 @@
 #define MAX_CONCURRENCY 1000
 
-/*
-enum METHOD {
-        GET,
-        SET
-};*/
-
-/*
-struct operation {
-        enum METHOD method;
-        char *key;
-        int key_len;
-        char *value;
-        int value_len;
-};*/
-
 struct pool_list {
 	int fd;
 	struct pool_list *next;
 };
 
-//void parse_message(char *message, int len, struct operation *operation);
-//void process_operation(struct operation *operation, char *message, int *len);
+// This is the hashtable on the server
+struct gwkv_server *ht;
```

Comment:
Why is this here and not in server-kv.c? I may just be being a dumb right now, but my gut says it should be in the .c.